### PR TITLE
feat(i18n): add `pl_PL` translation

### PR DESCRIPTION
**Pull request by Adamo2499**
Menu, sorting options, installed mods
image

Marketplace settings:
image

### CODE REVIEW

File: src/resources/locales/pl.json [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+      "Apps": "Aplikacje",
+      "Installed": "Zainstalowane"
+    },
+    "snippets": {
+      "addTitle": "Dodaj Snippet",
+      "duplicateName": "Ta nazwa jest już zajęta!",
+      "editTitle": "Edytuj Snippet",
+      "viewTitle": "Pokaż Snippet",
+      "customCSS": "Niestandardowy CSS",
+      "customCSSPlaceholder": "Wprowadź tutaj swój własny CSS! Można go znaleźć w zainstalowanej zakładce zarządzania.",
```
Comment (rxri) [inline src/resources/locales/pl.json:41]:
```suggestion
      "customCSSPlaceholder": "Wprowadź tutaj swój własny CSS! Możesz go znaleźć w zakładce 'Zainstalowane' dla zarządzania.",
```

File: src/resources/locales/pl.json [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+      "reloadNow": "Przeładuj teraz",
+      "reloadLater": "Przeładuj póżniej"
+    },
+    "backupModal": {
+      "title": "$t(settings.backupHeading)",
+      "settingsCopied": "Skopiowano do schowka",
+      "noDataPasted": "Dane nie zostały wklejone",
+      "invalidJSON": "Nieprawidłowy JSON",
+      "inputLabel": "Ustawienia Marketplace",
+      "inputPlaceholder": "Skopiuj / wklej tu swoje ustawienia",
```
Comment (rxri) [inline src/resources/locales/pl.json:64]:
```suggestion
      "inputPlaceholder": "Skopiuj/wklej tu swoje ustawienia",
```

File: src/resources/locales/pl.json [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+      "exportBtn": "Eksportuj",
+      "importBtn": "Importuj",
+      "fileImportBtn": "Importuj z pliku"
+    },
+    "devTools": {
+      "title": "Narzędzia developerskie do motywów",
+      "noThemeInstalled": "Błąd: Nie zainstalowano motywu",
+      "noThemeManifest": "Błąd: Nie znaleziono pliku manifestu motywu",
+      "colorIniEditor": "Edytor color.ini",
+      "colorIniEditorPlaceholder": "[your-colour-scheme-name]",
```
Comment (rxri) [inline src/resources/locales/pl.json:74]:
```suggestion
      "colorIniEditorPlaceholder": "[nazwa-twojego-koloru]",
```

File: src/resources/locales/pl.json [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+      "spicetifyMarketplace": "Spicetify Marketplace",
+      "newUpdate": "Nowa aktualizacja",
+      "addCSS": "Dodaj CSS",
+      "search": "Wyszukaj",
+      "installed": "Zainstalowane",
+      "lastUpdated": "Ostatnio zaktualizowane {{val, datetime}}",
+      "externalJS": "zewnętrzny JS",
+      "archived": "archiwalny",
+      "dark": "ciemny",
+      "light": "jasne",
```
Comment (rxri) [inline src/resources/locales/pl.json:97]:
```suggestion
      "light": "jasny",
```

File: src/resources/locales/pl.json [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+    },
+    "github": "GitHub",
+    "install": "Zainstaluj",
+    "remove": "Odinstaluj",
+    "save": "Zapisz",
+    "colour_one": "kolor",
+    "colour_other": "kolory",
+    "favourite": "ulubione",
+    "notifications": {
+      "wrongLocalTheme": "Ustaw current_theme w config-xpui.ini na \"marketplace\", aby zainstalować motywy za pomocą Marketplace.",
```
Comment (rxri) [inline src/resources/locales/pl.json:122]:
```suggestion
      "wrongLocalTheme": "Ustaw current_theme w config-xpui.ini na \"marketplace\", aby instalować motywy za pomocą Marketplace.",
```

File: src/resources/locales/pl.json [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+      "tooManyRequests": "Za dużo żądań, spokojnie",
+      "noCdnConnection": "Marketplace nie może połączyć się z CDN. Sprawdź swoją konfigurację internetową",
+      "markdownParsingError": "Error parsing markdown (HTTP {{status}})",
```
Comment (rxri) [inline src/resources/locales/pl.json:125]:
```suggestion
      "markdownParsingError": "Błąd podczas parsowania markdownu (HTTP {{status}})",
```

File: src/resources/locales/pl.json [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+      "title": "Ustawienia Marketplace",
+      "optionsHeading": "Opcje",
+      "starCountLabel": "Ilość gwiazdek",
+      "tagsLabel": "Tagi",
+      "showArchived": "Pokaż archiwalne repozytoria",
+      "devToolsLabel": "Narzędzia do tworzenia motywów",
+      "hideInstalledLabel": "Ukryj zainstalowane podczas przeglądania",
+      "colourShiftLabel": "Zmieniaj kolory co minutę",
+      "albumArtBasedColors": "Zmień kolory bazując na okładce albumu",
+      "albumArtBasedColorsMode": "Colour scheme (ColorApi) mode",
```
Comment (rxri) [inline src/resources/locales/pl.json:13]:
```suggestion
      "albumArtBasedColorsMode": "Tryb schematu kolorów (ColorApi)",
```

**Review by rxri — CHANGES_REQUESTED** [review]
Why did you use deepl for this if you're native polish speaker 🤦‍♀️ 
image

File: src/resources/locales/pl.json [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+      "tabsHeading": "Karty",
+      "resetHeading": "Zresetuj",
```
Comment (rxri) [inline src/resources/locales/pl.json:18]:
```suggestion
      "resetHeading": "Reset",
```

File: src/resources/locales/pl.json [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+      "resetBtn": "$t(settings.resetHeading)",
```
Comment (rxri) [inline src/resources/locales/pl.json:19]:
```suggestion
      "resetBtn": "Zresetuj",
```

**Review by rxri — CHANGES_REQUESTED** [review]
these two as well

**Review by theRealPadster — APPROVED** [review]

File: src/resources/locales/index.ts [hunk 10]
```diff
@@ -18,5 +19,6 @@ export default {
   "zh-TW": zhTW,
   "zh-CN": zhCN,
   et,
+  "pl-PL": pl,
```
Comment (rxri) [inline src/resources/locales/index.ts:22]:
```suggestion
  pl,
```
spotify is confusing me

**Review by rxri — APPROVED** [review]

**Pull request merged**